### PR TITLE
WF camera/ Updates

### PR DESCRIPTION
**Pull request by AArlauskas**
Name changes
Variable naming fixes
Saved Image file naming fixed
Default resoliution set to maximum
Output camera view is fixed to window size

### CODE REVIEW

**Comment by Pilkas-git** [comment]
Requested changes approved.

**Review by Pilkas-git — APPROVED** [review]
Code changed following requests.

File: Moodis/Moodis/cameraForm.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
+﻿using AForge.Video;
+using AForge.Video.DirectShow;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace moodie
+{
+    public partial class cameraForm : Form
```
Comment (tmaciulis22) [inline Moodis/Moodis/cameraForm.cs]: Class names should start with capital letters

File: Moodis/Moodis/cameraForm.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        }
+
+        private FilterInfoCollection webcam;
+        private VideoCaptureDevice cam;
+        private const string WarningMsg = "You must first turn on the camera!";
+
+        private void CameraFormLoad(object sender, EventArgs e)
+        {
+            webcam = new FilterInfoCollection(FilterCategory.VideoInputDevice);
+            foreach(FilterInfo VideoCaptureDevice in webcam)
```
Comment (tmaciulis22) [inline Moodis/Moodis/cameraForm.cs:23]: change FilterInfo to var

File: Moodis/Moodis/cameraForm.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            {
+                cmbOutputDevices.Items.Add(VideoCaptureDevice.Name);
+            }
+            cmbOutputDevices.SelectedIndex = 0;
+
+            cam = new VideoCaptureDevice(webcam[cmbOutputDevices.SelectedIndex].MonikerString);
+
+            foreach (VideoCapabilities option in cam.VideoCapabilities)
```
Comment (tmaciulis22) [inline Moodis/Moodis/cameraForm.cs]: same here, change to var

File: Moodis/Moodis/cameraForm.cs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+            saveFileDialog1.FileName = fileName;
+            try
+            {
+                if (saveFileDialog1.ShowDialog() == DialogResult.OK)
+                {
+                    picBox.Image.Save(saveFileDialog1.FileName);
+
+                }
+            }
+
```
Comment (tmaciulis22) [inline Moodis/Moodis/cameraForm.cs]: Empty line

File: Moodis/Moodis/cameraForm.cs [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        private void ComboBoxResoliution(object sender, EventArgs e)
+        {
+            if(cam.IsRunning)
+            {
+                cam.Stop();
+                cam.VideoResolution = cam.VideoCapabilities[cmbCamResoliution.SelectedIndex];
+                cam.Start();
+            }
+
```
Comment (tmaciulis22) [inline Moodis/Moodis/cameraForm.cs]: Empty line

**Review by tmaciulis22 — CHANGES_REQUESTED** [review]
Minor changes, but all in all good job 👍

File: Moodis/Moodis/Moodis.csproj [hunk 6]
```diff
@@ -0,0 +1,97 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{B5CEA781-5F92-4EDF-BC87-CF352F9AD6BF}</ProjectGuid>
+    <OutputType>WinExe</OutputType>
+    <RootNamespace>Moodie</RootNamespace>
+    <AssemblyName>Moodie</AssemblyName>
```
Comment (Pilkas-git) [inline Moodis/Moodis/Moodis.csproj:10]: Change both of "Moodie" to Moodis.

File: Moodis/Moodis/Program.cs [hunk 7]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace moodie
```
Comment (Pilkas-git) [inline Moodis/Moodis/Program.cs]: moodis not moodie. This is also going to edit a lot of references so be careful when renaming.

File: Moodis/Moodis/Properties/AssemblyInfo.cs [hunk 8]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("moodie")]
```
Comment (Pilkas-git) [inline Moodis/Moodis/Properties/AssemblyInfo.cs]: Change title to moodis.

File: Moodis/Moodis/Properties/AssemblyInfo.cs [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("moodie")]
```
Comment (Pilkas-git) [inline Moodis/Moodis/Properties/AssemblyInfo.cs]: Change to moodis.

File: Moodis/Moodis/cameraForm.cs [hunk 10]
```diff
@@ -0,0 +1,100 @@
+﻿using AForge.Video;
+using AForge.Video.DirectShow;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace moodie
```
Comment (Pilkas-git) [inline Moodis/Moodis/cameraForm.cs]: moodis not moodie. This is also going to edit a lot of references so be careful when renaming.

**Review by Pilkas-git — CHANGES_REQUESTED** [review]

File: Moodis/Moodis/cameraForm.cs (hunk 2 above)
Comment (Pilkas-git) [reply Moodis/Moodis/cameraForm.cs:23]: I don't believe he can change that based on the documentation www.aforgenet.com/framework/docs/html/8d13e608-64f7-8c03-84b1-1bce74dabecd.htm

**Comment by Pilkas-git** [comment]
Requested changed done in latest commit.

**Review by Pilkas-git — APPROVED** [review]
Completed all of the requested changes. Approved

**Review by tmaciulis22 — APPROVED** [review]

**Pull request merged**